extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1209,6 +1222,7 @@ public static function getCurrentStagingHost( $dbName='', $default='', $host = n
 	 * (preview/verify/sandbox).wikiname.wikia.com
 	 * en.wikiname.developer.wikia-dev.com
 	 * wikiname.developer.wikia-dev.com
+	 * @TODO MAIN-7994 support custom domains
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)